src/managers/world-manager.js

```diff
@@ -0,0 +1,84 @@
+'use strict';
+
+var WorldApi = require('../service/world-api-adapter');
+var RunManager =  require('./run-manager');
+var AuthManager = require('./auth-manager');
+// var _pick = require('../util/object-util')._pick;
+var worldApi;
+
+// var defaults = {
```

Comment:
Passing in 'files' to the WorldManager still doesn't feel right :-/ How about this?

```javascript

var wm = F.manager.WorldManager({
	account: 'x',
	project: 'y',
	transport: {
		onError: function () {
			console.log('Error with world manager');
		}
	},
	world: {
		transport: {
			onError: function () {
				console.log('Error with world service');
			}
		}
	},
	run: {
		files: []
	}
});

function WorldManager (options) {
	var baseOptions = $.extend(true, {}, options);
	delete baseOptions.run;
	delete baseOptions.world;

	var runOptions = $.extend(true, {}, baseOptions, options.run);
	var worldOptions = $.extend(true, {}, baseOptions, options.world);

	var runManager = new F.manager.Run(runOptions);
	var wa = worldApi = new WorldApi(worldOptions);
}

```